.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 14.19.1
```

Comment:
Do `nvm` automatically pick this up

Reply:
If you're using asdf, you can have it [read from .nvmrc anyway](https://github.com/asdf-vm/asdf-nodejs#nvmrc-and-node-version-support). nvm does not read from `.tool-versions` at all though. Unfortunately, the ideal solution would be to have both.

Reply:
Let's use `.nvmrc` if that works for both 🤷